content/docs/api-reference/app-registrations-api/index.md

```diff
@@ -74,24 +69,40 @@ The client secret ***must*** be taken note of at this point, as it will be redac
 }
 ```
 
+{{< caution >}}
+**Caution:** The expiry date refers to the secret only, any tokens generated with this secret will not automatically become invalid when the secret expires or is rotated. Each token has a TTL of 1 hour by default. 
```

Comment:
1 hour by default gives me the sense that I (as the customer) may be able to change that TTL. Could just remove "by default"

Reply:
Fixed :)